brion/plugin/compartmentReportBinary.cpp

```diff
@@ -166,10 +166,10 @@ void _initAIOControlBlock(aiocb& block, const AIOReadData& readData)
     block.aio_lio_opcode = LIO_READ;
 }
 
-void _readAsync(aiocb** operations, size_t length)
+void _readAsync(aiocb** operations, const size_t length)
 {
     if (lio_listio(LIO_WAIT, operations, length, nullptr) == -1)
-        throw std::runtime_error("Error in AIO operation setup" +
+        throw std::runtime_error("Error in AIO setup: " +
```

Comment:
LBTHROW outputs to stderr. I use throw when I don't want to pollute the console output, but I it doesn't look like it's the case here.